### PR TITLE
Clarify maximum timeout in Watchdog.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Clarify line about maximum timeout to resolve issue #1133

### CODE REVIEW

**Comment by AnotherButler** [comment]
ping @jamesbeyond @0xc0170

File: docs/porting/target/Watchdog.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Implement the ResetReason API when implementing the Watchdog API. The ResetReaso
 
 - Sleep and debug modes don't stop the watchdog timer from counting down.
 - The function `hal_watchdog_init` is safe to call repeatedly. The function's implementation must not do anything if `hal_watchdog_init` has already initialized the hardware watchdog timer.
-- Maximum supported timeout is `UINT32_MAX` milliseconds; minimum timeout is 1ms.
+- `UINT32_MAX` milliseconds is the maxinum allowed max_timeout `hal_watchdog_get_platform_features()` returns; minimum timeout is 1 ms.
```
Comment (0xc0170) [inline docs/porting/target/Watchdog.md]: `maxinum ` -> `maximum

**Review by 0xc0170 — CHANGES_REQUESTED** [review]
otherwise LGTM

File: docs/porting/target/Watchdog.md (hunk 1 above)
Comment (AnotherButler) [reply docs/porting/target/Watchdog.md]: 😱 Good catch

**Pull request merged**